public/app/percona/add-instance/components/AddRemoteInstance/AddRemoteInstance.service.tsx

```diff
@@ -172,8 +172,13 @@ export const toPayload = (values: any, discoverName?: string, type?: InstanceAva
     }
   }
 
-  if (type === Databases.mongodb && values.tls) {
-    data.authentication_mechanism = 'MONGODB-X509';
+  if (type === Databases.mongodb) {
+    if (values.tls) {
+      data.authentication_mechanism = 'MONGODB-X509';
+    }
+    if (values.disable_collectors) {
+      data.disable_collectors = values.disable_collectors.split(',').map((item: string) => item.trim());
```

Comment:
Should we really allow names with spaces? (e.g. `name 3`)

Reply:
It seems to me that this should remain on the conscience of the user. But if such a check is necessary, then it is done not here, but in the validation function. This line only helps the user not to pay attention to formatting. Initially, Carlos suggested setting a rule for the user, according to which he enters everything separated by commas, without spaces. (example: "name1,name2,name3"). Such writing is usually not familiar, so I added the trim function, which removes unnecessary spaces from the end and from the beginning so that the user can type more freely. (example: " name1,name2,     name3, name4). The last example will work fine.

Reply:
![image](https://user-images.githubusercontent.com/90199600/143828538-b087a1a1-26bd-4ec4-bceb-04666ac553e9.png)

Reply:
As for the spaces in the names of collectors, as an example, Carlos called the collector "index stats" and wrote it with a space. I'll check with him just in case this moment and whether we need to process it. 
![image](https://user-images.githubusercontent.com/90199600/143830883-7cbbd2b4-9efd-4019-bc16-db6a149d5788.png)

Reply:
@percona-csalguero, can you help us?

Reply:
there should be no spaces. index stats is actually `indexStats`:
https://github.com/percona/mongodb_exporter/blob/main/README.md
https://docs.mongodb.com/manual/reference/operator/aggregation/indexStats/

Reply:
`--mongodb.indexstats-colls`  so just `indexstats`

Reply:
@denisok, thanks. We already talked with Carlos, and I have already added a new field and function that removes all spaces